setup.py

```diff
@@ -325,8 +326,16 @@ def check_file(f):
 
     # Use copies instead of symbolic files.
     # Windows has very poor support for them.
-    sym_files = ['tools/shared/_utils_internal.py']
-    orig_files = ['torch/_utils_internal.py']
+    sym_files = [
+        'tools/shared/_utils_internal.py',
+        'torch/utils/benchmark/utils/valgrind_wrapper/callgrind.h',
+        'torch/utils/benchmark/utils/valgrind_wrapper/valgrind.h',
```

Comment:
Are you sure these files will get packaged correctly when we make Python wheels?

Reply:
In particular, I don't see any update to `package_data`

Reply:
They aren't. I mistakenly took the entry out when I removed the symlinks; I'll put it back.